be1-go/channel/election/mod.go

```diff
@@ -681,6 +684,70 @@ func gatherResults(questions map[string]*question,
 	return resultElection, nil
 }
 
+// getVoteIndex extracts the index of the vote from the validVotes
+func (c *Channel) getVoteIndex(vote validVote) (int, error) {
+	switch m := c.electionType; m {
+	// open ballot votes have the index in plain text
+	case messagedata.OpenBallot:
+		index, _ := vote.index.(int)
```

Comment:
what if the cast fails ? You should check :

```go
index, ok := ...
if !ok {
  return xerrors....
```

Reply:
same bellow

Reply:
I already check during the verification of the cast_votes, should I check it here too ?